pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+markers =
```

Comment:
do we need to register any of these markers as well?
https://github.com/NVIDIA-Merlin/models/blob/e7fe759c7628e3902e029b9930e6b155f478ef25/tests/conftest.py#L86-L110